Day-I-Social-Card/social-card/src/index.css

```diff
@@ -3,3 +3,7 @@ body {
   padding: 0.4rem;
   font-family: sans-serif;
 }
+
+.root{
```

Comment:
you have two CSS files that try to reference the root React element. What is the difference between the two?